azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Improve quest Fields of Grief (part2)

## Changes Proposed:
-  Increase spawn locations from 10->32.
-  Add them to a pool, 20 spawns at once (easy to change this)
- Respawn time lowered from 5->2 minutes


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4701



## Tests Performed:
- Applied SQL ~~, not tested in game~~
- Tested in game



## How to Test the Changes:

- `.quest add 365`
- `.tele sollidenf`
- loot pumpkins, notice respawn is 2 minutes and the spawns are more numerous and in more locations than before

## Known Issues and TODO List:

- This was already discussed in: https://github.com/azerothcore/azerothcore-wotlk/pull/4529 where these suggested changes where found wrong.
- This is not a "purist" change, as it would be more "Classic" than "Vanilla". Opinions regarding this might differ - this is merely a suggestion.
- For what it's worth, this is closer to TC than what we have now.


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
